laurent22/joplin
# Desktop: Resolves #9193: Upgrade CodeMirror dependencies

# Summary

- Updates CodeMirror dependencies in `packages/editor`
- Removes an unused CodeMirror dependency of `packages/app-mobile`

Resolves #9193.

# Testing

Desktop:
0. Enable the beta editor
1. Enable Vim mode
2. Verify that `/` find works
3. Move with `h`/`j`/`k`/`l`
4. Disable Vim mode
5. Paste a screenshot into the editor
6. Drag and drop a note into the editor
7. Paste `https://en.m.wikipedia.org/wiki/Duck` into the editor (verify it highlights as a link)

Tested on Ubuntu 23.10.

Mobile:
1. Write `This is a test...`
2. Select `test` and click "bold"
3. Press the indent button to give the just-written text code formatting.
4. Create a three-item numbered list with the "toggle list" button
5. Indent the second item
6. Verify that the list re-numbered
8. De-indent the second item

Tested on Android 7.
